src/transcoding.ts

```diff
@@ -309,7 +351,15 @@ export function transcode(
     }
   }
   // request is supposed to have keys in camelCase.
-  const snakeRequest = requestChangeCaseAndCleanup(request, camelToSnakeCase);
+  let fieldsToChange = undefined;
+  if (requestFields) {
```

Comment:
Let's make `requestFields` required. I think `transcode()` is only called from [here](https://github.com/googleapis/gax-nodejs/blob/a5702bd19ce7daae7fee5b60202bbf834e0621b0/src/fallbackRest.ts#L62), and it's completely reasonable to assume this parameter is always required.

`transcode` is not exported publicly so it's an internal change that won't be breaking.

Reply:
This would break a lot of tests, so it implies a different expected behavior. Is there a specific reason you want to make it required?

Reply:
Nope, just to save on some `if` statements. It's ok to leave it as is.